processing file asmFiles/branchforward.asm (1 of 41)
processing file asmFiles/branchLost.asm (2 of 41)
processing file asmFiles/count_days.asm (3 of 41)
processing file asmFiles/count_days.asm~ (4 of 41)
processing file asmFiles/fib.asm (5 of 41)
processing file asmFiles/forwarding_test.asm (6 of 41)
--- memsim.hex	2014-03-03 14:34:09.227195529 -0500
+++ memcpu.hex	2014-03-03 14:34:09.210195079 -0500
@@ -26,5 +26,4 @@
 :040FFF000000002FBF
 :041000000000004E9E
 :041001000000000FDC
-:041002000000006D7D
 :00000001FF
processing file asmFiles/Hanye_count_days.asm (7 of 41)
processing file asmFiles/Hanye_mult.asm (8 of 41)
processing file asmFiles/Hanye_mult_procedure.asm (9 of 41)
processing file asmFiles/hanye_test1.asm (10 of 41)
--- memsim.hex	2014-03-03 14:34:15.291355884 -0500
+++ memcpu.hex	2014-03-03 14:34:15.258355011 -0500
@@ -15,6 +15,6 @@
 :04000E0003E0000803
 :04000F000C000006DB
 :0400100008000005DF
-:0400190000000002E1
-:0400320000000002C8
+:0400190000000001E2
+:0400320000000001C9
 :00000001FF
processing file asmFiles/hanye_test2.asm (11 of 41)
processing file asmFiles/jumpbranch.asm (12 of 41)
processing file asmFiles/kevinforward.asm (13 of 41)
processing file asmFiles/lui.asm (14 of 41)
processing file asmFiles/lwbranchcycletest.asm (15 of 41)
processing file asmFiles/lwsw.asm (16 of 41)
processing file asmFiles/lwuse.asm (17 of 41)
processing file asmFiles/mult.asm (18 of 41)
processing file asmFiles/mult.asm~ (19 of 41)
processing file asmFiles/mult_procedure2.asm (20 of 41)
processing file asmFiles/mult_procedure.asm (21 of 41)
processing file asmFiles/mult_procedure.asm~ (22 of 41)
processing file asmFiles/search.asm (23 of 41)
processing file asmFiles/test.branch.asm (24 of 41)
processing file asmFiles/test.branchjump1.asm (25 of 41)
processing file asmFiles/test.jr.asm (26 of 41)
processing file asmFiles/test.jump.asm (27 of 41)
processing file asmFiles/test.ldst1.asm (28 of 41)
processing file asmFiles/test.ldst1.asm~ (29 of 41)
processing file asmFiles/test.ldst2.asm (30 of 41)
processing file asmFiles/test.ldst2.asm~ (31 of 41)
processing file asmFiles/test.ldst3.asm (32 of 41)
processing file asmFiles/test.loadstore.asm (33 of 41)
processing file asmFiles/test.rtype2.asm (34 of 41)
processing file asmFiles/test.rtype3(j).asm (35 of 41)
processing file asmFiles/test.rtype.asm (36 of 41)
processing file asmFiles/test.simplebranch.asm (37 of 41)
processing file asmFiles/test.slt.asm (38 of 41)
processing file asmFiles/y.test.ldst3.asm (39 of 41)
processing file asmFiles/z.fib.asm (40 of 41)
processing file asmFiles/z.search.asm (41 of 41)
processing file asmFiles/branchforward.asm (1 of 44)
processing file asmFiles/branchLost.asm (2 of 44)
processing file asmFiles/count_days.asm (3 of 44)
processing file asmFiles/count_days.asm~ (4 of 44)
processing file asmFiles/fib.asm (5 of 44)
processing file asmFiles/forwarding_test.asm (6 of 44)
--- memsim.hex	2014-03-03 14:37:03.491804149 -0500
+++ memcpu.hex	2014-03-03 14:37:03.436802695 -0500
@@ -26,5 +26,4 @@
 :040FFF000000002FBF
 :041000000000004E9E
 :041001000000000FDC
-:041002000000006D7D
 :00000001FF
processing file asmFiles/forwarding_test.asm~ (7 of 44)
--- memsim.hex	2014-03-03 14:37:05.031844882 -0500
+++ memcpu.hex	2014-03-03 14:37:05.008844273 -0500
@@ -26,5 +26,4 @@
 :040FFF000000002FBF
 :041000000000004E9E
 :041001000000000FDC
-:041002000000006D7D
 :00000001FF
processing file asmFiles/Hanye_count_days.asm (8 of 44)
processing file asmFiles/Hanye_mult.asm (9 of 44)
processing file asmFiles/Hanye_mult_procedure.asm (10 of 44)
processing file asmFiles/hanye_test1.asm (11 of 44)
--- memsim.hex	2014-03-03 14:37:11.095005249 -0500
+++ memcpu.hex	2014-03-03 14:37:11.059004298 -0500
@@ -15,6 +15,6 @@
 :04000E0003E0000803
 :04000F000C000006DB
 :0400100008000005DF
-:0400190000000002E1
-:0400320000000002C8
+:0400190000000001E2
+:0400320000000001C9
 :00000001FF
processing file asmFiles/hanye_test1.asm~ (12 of 44)
--- memsim.hex	2014-03-03 14:37:12.586044686 -0500
+++ memcpu.hex	2014-03-03 14:37:12.567044184 -0500
@@ -15,6 +15,6 @@
 :04000E0003E0000803
 :04000F000C000006DB
 :0400100008000005DF
-:0400190000000002E1
-:0400320000000002C8
+:0400190000000001E2
+:0400320000000001C9
 :00000001FF
processing file asmFiles/hanye_test2.asm (13 of 44)
processing file asmFiles/hanye_test2.asm~ (14 of 44)
processing file asmFiles/jumpbranch.asm (15 of 44)
processing file asmFiles/kevinforward.asm (16 of 44)
processing file asmFiles/lui.asm (17 of 44)
processing file asmFiles/lwbranchcycletest.asm (18 of 44)
processing file asmFiles/lwsw.asm (19 of 44)
processing file asmFiles/lwuse.asm (20 of 44)
processing file asmFiles/mult.asm (21 of 44)
processing file asmFiles/mult.asm~ (22 of 44)
processing file asmFiles/mult_procedure2.asm (23 of 44)
processing file asmFiles/mult_procedure.asm (24 of 44)
processing file asmFiles/mult_procedure.asm~ (25 of 44)
processing file asmFiles/search.asm (26 of 44)
processing file asmFiles/test.branch.asm (27 of 44)
processing file asmFiles/test.branchjump1.asm (28 of 44)
processing file asmFiles/test.jr.asm (29 of 44)
processing file asmFiles/test.jump.asm (30 of 44)
processing file asmFiles/test.ldst1.asm (31 of 44)
processing file asmFiles/test.ldst1.asm~ (32 of 44)
processing file asmFiles/test.ldst2.asm (33 of 44)
processing file asmFiles/test.ldst2.asm~ (34 of 44)
processing file asmFiles/test.ldst3.asm (35 of 44)
processing file asmFiles/test.loadstore.asm (36 of 44)
processing file asmFiles/test.rtype2.asm (37 of 44)
processing file asmFiles/test.rtype3(j).asm (38 of 44)
processing file asmFiles/test.rtype.asm (39 of 44)
processing file asmFiles/test.simplebranch.asm (40 of 44)
processing file asmFiles/test.slt.asm (41 of 44)
processing file asmFiles/y.test.ldst3.asm (42 of 44)
processing file asmFiles/z.fib.asm (43 of 44)
processing file asmFiles/z.search.asm (44 of 44)
